src/strings/csv/en.csv

```diff
@@ -318,6 +318,8 @@ DESTINATION_REQUIRED,Destination *,
 DETAILED_SITE,Detailed Site,
 DETAILS,Details,
 DIRECTION_OR_DESCRIPTION,Direction or Description,
+DOCUMENT_LIMIT_REACHED,Document Limit Reached,
+DOCUMENT_LIMIT_REACHED_MESSAGE,You may only upload {0} documents. Talk to your Terraformation representative to have them remove files if you would like to upload more.,
```

Comment:
Oops missed this, we won't be removing files for now. Not sure that is decided yet. TF representative can upload more files on their behalf. Maybe just `Talk to your Terraformation representative if you would like to upload more.`?

Reply:
I also thought this string wasn't very well considered. Good call, will update 👍